.changeset/mean-dingos-check.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/unenv-preset": patch
+---
+
+fix: avoid circular reference to aliased import in Performance polyfill
```

Comment:
This should be removed if the change is no more needed (with the update to RC 14)

Reply:
Removed.